oarepo_ui/translations/cs/LC_MESSAGES/messages.po

```diff
@@ -534,3 +534,105 @@ msgstr ""
 #: /Users/m/w/cesnet/oarepo-ui/oarepo_ui/templates/oarepo_ui/header_frontage.html:26
 msgid "Type and press enter to search"
 msgstr "Zadejte a stiskněte Enter pro vyhledávání"
+
+msgid "Community website"
+msgstr "Webové stránky komunity"
+
+msgid "Community selection"
+msgstr "Výběr komunity"
+
+msgid "Please select community in which your work will be published:"
+msgstr "Vyberte prosím komunitu, ve které bude vaše práce publikována:"
+
+msgid "All records must belong to a community."
+msgstr "Všechny záznamy musí patřit komunitě."
+
+msgid "Go back"
```

Comment:
Back -> Zpátky

Reply:
@mirekys Hi, I replaced "go back" with "Home page, because I was using window object to basically pop the last entry and return to the last one (in a sense to return you to the page where you were previously), but I have come to realize that if you visit a form directly, by for example just pasting /_new url into the browser, when you click back it takes you back to your starting page i.e. google search, so I thought this was not ideal.